geode-core/src/test/java/org/apache/geode/test/dunit/rules/DistributedRestoreSystemProperties.java

```diff
@@ -51,7 +51,11 @@ public void before() throws Throwable {
       public void run() {
         if (originalProperties == null) {
           originalProperties = getProperties();
-          setProperties(new Properties(originalProperties));
```

Comment:
This line is identical to what's in the System Rules RestoreSystemProperties java class. How does this line fail? If there's a bug then we need to send a PR to the System Rules project for [RestoreSystemProperties.java](https://github.com/stefanbirkner/system-rules/blob/master/src/main/java/org/junit/contrib/java/lang/system/RestoreSystemProperties.java).